msal/tests/Test.MSAL.NET.Unit/RequestsTests/InteractiveRequestTests.cs

```diff
@@ -211,6 +211,8 @@ public void NoCacheLookup()
                 anEvent[EventBase.EventNameKey].EndsWith("ui_event") && anEvent[UiEvent.UserCancelledKey] == "false"));
             Assert.IsNotNull(_myReceiver.EventsReceived.Find(anEvent =>  // Expect finding such an event
                 anEvent[EventBase.EventNameKey].EndsWith("api_event") && anEvent[ApiEvent.UiBehaviorKey] == "select_account"));
+            Assert.IsNotNull(_myReceiver.EventsReceived.Find(anEvent =>  // Expect finding such an event
+                anEvent[EventBase.EventNameKey].EndsWith("ui_event") && anEvent[UiEvent.AccessDeniedKey] == "false"));
```

Comment:
Added `UiEvent.AccessDeniedKey` to preexisting unit tests

Reply:
are there any real unit tests for this class? It should be relatively simple to validate.
also seems like this integration test only validated that it's false (not set). Ideally the integration testing would also show an error situation.

---
In reply to: [200799705](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1114#discussion_r200799705) [](ancestors = 200799705)

Reply:
I added in more test coverage for the uiEvent I added. There are Telemetry tests in core.